<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundWindow</class>
 <widget class="QWidget" name="BackgroundWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>946</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>110</height>
       </size>
      </property>
      <property name="title">
       <string>Config</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="3">
          <widget class="QGroupBox" name="color2GroupBox">
           <property name="title">
            <string>Color2</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="Widgets::ColorPushButton" name="color2Button">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Style</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="styleCombo">
           <item>
            <property name="text">
             <string>One Color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Two Colors</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Backgroud Image</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QGroupBox" name="imageFileGroupBox">
           <property name="title">
            <string>Image File</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="browseButton">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="fileEdit">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGroupBox" name="color1GroupBox">
           <property name="title">
            <string>Color</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="Widgets::ColorPushButton" name="colorButton">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Superquadric</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Theta Resolution</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Theta Roundness</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="thetaResolutionSpinBox"/>
              </item>
              <item row="1" column="3">
               <widget class="QDoubleSpinBox" name="phiRoundnessSpinBox">
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QDoubleSpinBox" name="thetaRoundnessSpinBox">
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="phiResolutionSpinBox"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Phi Resolution</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Phi Roundness</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Thickness</string>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QDoubleSpinBox" name="thicknessSpinBox">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="minimum">
                 <double>0.000100000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.000100000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Shape</string>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QComboBox" name="shapeComboBox">
                <item>
                 <property name="text">
                  <string>Ellipsoidal</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Toroidal</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="6">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Preview</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="previewLayout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::ColorPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">colorpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
